docker/Dockerfile_hadoop

```diff
@@ -91,6 +89,7 @@ RUN wget https://archive.apache.org/dist/spark/spark-$SPARK_VERSION/spark-$SPARK
     && cp $HIVE_HOME/conf/hive-site.xml $SPARK_HOME/conf \
     && cp $SPARK_HOME/yarn/*.jar $HADOOP_HOME/share/hadoop/yarn/lib
 RUN cp $HIVE_HOME/lib/mysql-connector-java-5.1.24.jar $SPARK_HOME/jars
+RUN cp $HBASE_HOME/lib/hbase-protocol-1.1.2.jar $SPARK_HOME/jars
```

Comment:
Wht should we add this line?

Reply:
when build with spark, it needs this hbase-protocol.jar, otherwise it throws 'NoClassDefFoundError: org/apache/hadoop/hbase/protobuf/generated/HFileProtos'.